classes/task/check_objects_location.php

```diff
@@ -44,8 +47,6 @@ public function get_name() {
      * Execute task
      */
     public function execute() {
-        manipulator::setup_and_run_object_manipulator('\\tool_objectfs\\local\\object_manipulator\\checker');
+        (new manipulator_builder())->build(checker::class)->execute();
```

Comment:
This line is repeated in 5 tasks can we abstract this away so they only need to set their associated class.

Reply:
I added the abstract class  :)